project-base/kubernetes/ingress.yml

```diff
@@ -2,6 +2,8 @@ apiVersion: extensions/v1beta1
 kind: Ingress
 metadata:
     name: shopsys
+    annotations:
+        nginx.ingress.kubernetes.io/proxy-body-size: "32m"
```

Comment:
should it be like string?
in https://github.com/kubernetes/ingress-nginx/blob/master/docs/user-guide/nginx-configuration/annotations.md#custom-max-body-size they don't use quotes.